packages/node/src/methods/app-instance/propose-install-virtual/operation.ts

```diff
@@ -92,13 +91,12 @@ export async function getOrCreateStateChannelBetweenVirtualAppParticipants(
     stateChannel = await StateChannel.getStateChannelWithOwners(
       initiatorXpub,
       responderXpub,
+      network,
       store
     );
   } catch (e) {
     if (
-      e
-        .toString()
-        .includes(NO_CHANNEL_BETWEEN_NODES(initiatorXpub, responderXpub)) &&
+      e.toString().includes("Call to getStateChannel failed") &&
```

Comment:
Remove the use of string literals when its value is expected to appear in multiple places, makes comparison less error-prone.